app/src/main/java/woowacourse/paint/model/DrawingHistory.kt

```diff
@@ -0,0 +1,11 @@
+package woowacourse.paint.model
+
+class DrawingHistory {
+    private val _history = mutableListOf<DrawingElement>()
+    val history: List<DrawingElement>
+        get() = _history
```

Comment:
```suggestion
        get() = _history.toList()
```
다음과 같이 방어적 복사를 적용해보면 어떨까요??